units/initrd-parse-etc.service.in

```diff
@@ -8,17 +8,17 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Reload Configuration from the Real Root
+Description=Mountpoints Configured in the Real Root
+AssertPathExists=/etc/initrd-release
+
 DefaultDependencies=no
 Requires=initrd-root-fs.target
 After=initrd-root-fs.target
+
 OnFailure=emergency.target
 OnFailureJobMode=replace-irreversibly
-AssertPathExists=/etc/initrd-release
 
 [Service]
 Type=oneshot
-ExecStartPre=-systemctl daemon-reload
```

Comment:
BTW, does this line effectively run systemd-fstab-generator? What about the other generators?
IOW, running only systemd-fstab-generator is enough after this PR?

Reply:
I see. Thanks.